app/src/components/main/analytics/events/filtersPageEvents.js

```diff
@@ -36,11 +39,17 @@ export const getAddEditFilterModalEvents = (isEditMode) => {
       action: `Click on button Cancel in Modal ${modalType} Filter`,
       label: `Close Modal ${modalType} Filter`,
     },
-    clickOkBtn: {
-      category: FILTERS_PAGE,
-      action: `Click on button ${actionType} in Modal ${modalType} Filter`,
-      label: `${actionType} filter in Modal ${modalType} Filter`,
-    },
+    clickOkBtn: isEditMode
```

Comment:
Do we really need it here?
I mean left-hand expression of the ternary operator?

Reply:
We have GA4 analytics only for Add filter modal OK button, so I think yes, we need it

Reply:
In case if isEditMode === true we do not send anything.
If we want to reuse clickOkBtn in edit modal f.e. We can rewrite it to : 
<img width="528" alt="Screenshot 2023-03-17 at 3 59 09 PM" src="https://user-images.githubusercontent.com/84175555/225886690-ec5832c2-f7fc-400a-bde6-91f222cc037b.png">

Reply:
Sorry, not `isEditModa ? 'add' : 'edit'_modal` but `isEditModa ? 'add' : 'edit'_filter`

Reply:
According to the analytics dos I see that we decided to separate events for this modal between different categories (launches & filters).
So I suggest to parametrize function `getAddEditFilterModalEvents` with `category` parameter and move it upper to common place. In this case we will reuse the events for this modal across different pages.
Also may be provide events with necessary `category` and `editMode` to the modal inside of `showModalAction` function where it is called? As I see in the code it is clear where we need particular event type.

`LaunchFiltersToolbar`
![image](https://user-images.githubusercontent.com/31861805/225916094-e5e44614-62a4-45c4-98c3-ea52272b58ef.png)

![image](https://user-images.githubusercontent.com/31861805/225916233-7a8dd5f9-a640-4ca6-9b15-b1052344ca4e.png)

`FiltersPage`
![image](https://user-images.githubusercontent.com/31861805/225916353-4263badb-f818-4aec-b738-cc8c70f3054e.png)

@evgenijkorob @Bam6ycha

Reply:
Sorry, it's not about your question.
As now we don't track the event for `add modal` I suppose the @evgenijkorob solution is applicable.
When these modal events are migrated to GA4, they will be adjusted accordingly.

Reply:
Decided to do it in the scope of the next task connected to filter modal events.